### PR TITLE
Create basic prototype for the HTML Component

### PR DESCRIPTION
**Pull request by alex-deaconu**
## Description
Create basic prototype for the HTML Component

[stage-2]

## Motivation and Context
HTML Component Epic

## How Has This Been Tested?
Tested changes locally and on staging. Will add unit tests and polish the implementation in subsequent PRs.

## Release Plan:
- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed. 
- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
No

### CODE REVIEW

**Comment by alex-deaconu** [comment]
@ezequielc initial prototype for this. Please review. Thanks!

File: src/scripts/template-editor/components/directives/dtv-component-html.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        templateUrl: 'partials/template-editor/components/component-html.html',
+        link: function ($scope, element) {
+          $scope.factory = templateEditorFactory;
+
+          function _load() {
+            var html = $scope.getAvailableAttributeData($scope.componentId, 'html');
+
+            $scope.html = html;
+
+            $timeout(function () {
```
Comment (ezequielc) [inline src/scripts/template-editor/components/directives/dtv-component-html.js:18]: @alex-deaconu Curious, why do we need this event? Thanks!

**Review by ezequielc — APPROVED** [review]
@alex-deaconu Looks good 👍 
Just one clarification question inline. Thanks!

File: src/scripts/template-editor/components/directives/dtv-component-html.js (hunk 1 above)
Comment (alex-deaconu) [reply src/scripts/template-editor/components/directives/dtv-component-html.js:18]: @ezequielc no idea. I copied this from text; I will remove in my next PR. Thanks.

Comment (ezequielc) [reply src/scripts/template-editor/components/directives/dtv-component-html.js:18]: @alex-deaconu Sounds good 👍

**Pull request merged**